system/jhtree/jhtree.cpp

```diff
@@ -1924,6 +1988,33 @@ bool CKeyCursor::_ltEqual(IContextLogger *ctx)
         }
     }
     if (!lwm)
+    {
+        while (depth)
+        {
+            depth--;
+            if (depth < maxParentNodes)
+            {
+                node.set(parents[depth]);
+                if (node)
+                {
+                    unsigned numParentKeys = node->getNumKeys();
+                    unsigned parentNodeKey = parentNodeKeys[depth];
+                    if (parentNodeKey < numParentKeys-1)
+                    {   
+                        int rc = node->compareValueAt(recordBuffer, numParentKeys-1);
+                        if (rc < 0)
+                        {
+                            lwm = parentNodeKey+1;
+                            printf("Still in parent node at level %u %u\n", depth, numParentKeys);
```

Comment:
needs removing or putting on a trace flag that disables off (same for other tracing above)

Reply:
I will remove all the printf lines whether commented out or not.